wheel/src/api.rs

```diff
@@ -537,11 +537,11 @@ pub fn get_spends_for_trusted_block<'a>(
         .collect::<Vec<&'a [u8]>>();
 
     let output =
-        py.allow_threads(|| get_coinspends_for_trusted_block(constants, &generator, &refs, flags))?;
+        py.detach(|| get_coinspends_for_trusted_block(constants, &generator, &refs, flags))?;
 
     let dict = PyDict::new(py);
     dict.set_item("block_spends", output)?;
-    Ok(dict.into())
+    Ok(dict.into_any().unbind())
```

Comment:
I'm not really a pybind expert, but I would expect we could remove the call to `.unbind()` if we changed the return type to `Bound<PyAny>`. Which would save on a conversion, I think.